style/style.css

```diff
@@ -87,6 +85,268 @@ header a {
   background-image: url(/images/illustration-header-mobile-1.png);
   background-size: cover;
   background-position: center;
-  width: 190px;
-  height: 270px;
+  width: 147px;
+  height: 409px;
+}
+
+.portfolio-section {
+  width: 100%;
+}
+
+.content {
+  display: grid;
+  grid-template-columns: 1fr;
+  align-items: center;
+  font-family: "Inter", sans-serif;
+  height: 50rem;
+  margin-top: 50rem;
+}
```

Comment:
- [ ] As stated by the previous ◀️ ◀️  reviewer please 🙏 align the` work section` 🗒️ using the `display: grid;` and the content inside 📨  using `flexbox` as it is the [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#project--requirements) of the project. 


> In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.

  The purpose 🤓 of this exercise 🏃  is to teach 🧑‍🏫 you how to layout contents using `Grid` and `Flexbox` both. Also please remember that the `work section` includes all the part form `My recent works` upto the last project card. 

   In your case section `portfolio-section` and `project` section both should be inclosed in one section tag and be aligned using grid. The content inside of them like for example tech stack, project heading, description, image button etc, everything inside the the card,  they should be aligned using `flexbox`

Reply:
fixed layout contents using Grid and Flexbox